src/amo/css/OverallRating.scss

```diff
@@ -0,0 +1,40 @@
+.OverallRating {
+
+  form {
+    display: flex;
+    justify-content: center;
+
+    input {
+      appearance: none;
+      opacity: 0;
+      width: 0;
+      height: 0;
+    }
+
+    label {
```

Comment:
doesn't my initial `.OverallRating {}` block accomplish the same thing though? The style engine will only be looking at the tree of elements starting at `<div class="OverallRating" />`